src/main/java/seedu/momentum/model/project/CompletionStatus.java

```diff
@@ -0,0 +1,71 @@
+package seedu.momentum.model.project;
+
+/**
+ * Represents a Project's Completion status in the project book
```

Comment:
Can put {@code Project} here?

Reply:
There is no reference to the Project class in CompletionStatus thou